src/lib/stores/accountsStore.ts

```diff
@@ -1,28 +1,30 @@
-import { writable, type Writable } from 'svelte/store';
+import { writable } from 'svelte/store';
 import { ApiPromise, Keyring } from '@polkadot/api';
 import type { web3Enable, web3Accounts } from '@polkadot/extension-dapp';
 import type { AccountMap, MetaMap } from '$lib/polkadotApi';
 import { getMsaInfo } from '$lib/polkadotApi';
-import { Network, networkToInfo } from '$lib/stores/networksStore';
 import { isFunction } from '@polkadot/util';
+import type { NetworkInfo } from './networksStore';
 
-//All accounts
-export const storeValidAccounts = writable({});
-//Only provider accounts
-export const storeProviderAccounts = writable({});
+// All accounts
+export const storeValidAccounts = writable<string[]>([]);
+// Only provider accounts
+export const storeProviderAccounts = writable<string[]>([]);
+
+export const storeSelectedAccount = writable<string>('');
```

Comment:
I believe we're storing the current user as $transactionSigningAddress. Maybe worth going in and deleting $transactionSigningAddress in favor of $storeSelectedAccount.

Reply:
I think we should revisit the whole model but in a different issue and PR.